### DISCUSSION TITLE
Le lapin domestique

### DISCUSSION THREAD

**Discussion by evefevrier** [integration2/quizTIM2023#37] [Q&A]
Commentaires sur ce quiz

**Accès**
- Serveur de développement (protégé) :  https://timunix3.csfoy.ca/~quiz-TIM/captainfil/index.html
- Dépôt Github (privé pour le moment) : https://github.com/captainfil/quiz-tim
- Github.io :

**Comment by ArtificialOxygen** [comment]
Écrit par Lorie-Anne Côté

# Retour sur ton quiz "Le lapin domestique, connaitre son compagnon"

## 3 points forts

- Design sobre et épuré. Les couleurs sont bien choisies et me rappelle les couleurs du côté plus festif et commercial de la fête de Pâques justement associée aux lapins. Le design de ton quiz parait moderne et il est très plaisant à l’œil. 

- Boutons radio généralement fonctionnels. L’état survol de tes boutons radio est très original ! L’état sélectionné est bien visible : on comprend qu’elle réponse a été cochée. Le fait que les boutons radio soient stylés différemment à chaque question (avec des petits émoticônes, avec des images ou avec juste du texte) donne un style unique au quiz.

- Page fluide. Selon moi, ton quiz a une très bonne adaptation en version mobile et table. De plus, en version table, il n'y a aucun besoin de "scroller" toute la page pour pouvoir voir tout le contenu du quiz. Les éléments sont très bien alignés et centrés.

## 3 points à améliorer

- Il aurait été plutôt intéressant de savoir si nos choix de réponse étaient bons ou mauvais, puisqu'il n'y a pas de rétroactions (peut-être que tu n'as juste pas eu le temps de les compléter) donc ça se comprend.

- Attention, je crois que tu as oublié de changer le contenu de la balise title "Titre du quiz - Quiz Tim" du head dans ton document HTML (ne t'inquiète pas, tu n'es pas le seul).

- Je crois qu'il n'est pas possible de naviguer sur tes boutons radio à l'aide de la touche TAB et des flèches.

##  En bref

Tu peux être fier de ton travail ! J'ai bien aimé tester mes connaissances sur les lapins 😄

**Reply by phoenix-four** [reply]
Bon travail sur ton quizz!

Premièrement, je constate que je ne connais strictement rien aux lapins!

Design: J'aime bien le côté simple et épuré. Bonne utilisation des couleurs pastel qui donne un côté Pâques amusant.

Intégration: Fonctionnement très clair et bien fonctionnel! On comprend bien les rétroactions. :) Aussi, bon fonctionnement du responsive.

**Comment by evefevrier** [comment]
@captainfil 
Ce quiz est fantastique. Wow. 🥇
